src/common/sidebar/containers/SidebarContainer.tsx

```diff
@@ -48,13 +49,29 @@ export function SidebarContainer({ searchQuery, term }: SidebarContainerProps):
     term: term as Term,
     queryParams: { in_session: filter },
   });
+  // re assign the type of subjects so inSession property is available
+  const inSessionSubjects: InSessionSubject[] | null = subjects;
```

Comment:
This isn't required since you're mapping against the original thing. You can just return a `InSessionSubject` from the `map` below.